pytest_fixtures/component/taxonomy.py

```diff
@@ -89,14 +91,16 @@ def module_sca_manifest_org(module_org, module_sca_manifest, module_target_sat):
     return module_org
 
 
-@pytest.fixture(scope='function')
-def function_entitlement_manifest_org(function_org, function_entitlement_manifest, target_sat):
+@pytest.fixture
+def function_entitlement_manifest_org(function_entitlement_manifest, target_sat):
     """Creates an organization and uploads an entitlement mode manifest generated with manifester"""
-    target_sat.upload_manifest(function_org.id, function_entitlement_manifest.content)
-    return function_org
+    org = target_sat.api.Organization().create()
```

Comment:
Why did you decide to not use `function_org` fixture? Same for the `module_org` in the `module_entitlement_manifest_org` fixture.

https://github.com/SatelliteQE/robottelo/blob/c25b79fd0507c57bffc172c5a01cc1d4e01f1501/pytest_fixtures/component/taxonomy.py#L23-L25

Reply:
I was quite sure I had a good reason for doing this yesterday, but I'm unable to produce that reason now. I'll change both back to using the respective `org` fixtures.

Reply:
Actually, I may have spoken too soon - I need to run some additional tests and I'll report back definitively later today.